### PR TITLE
Fix flaky telemetry operator test

### PR DESCRIPTION
**Pull request by lindnerby**
**Description**

Changes proposed in this pull request:

- Increase timeout for certain tests
- Refactor mockserver creation
- Refactor tests

Related test-infra PR:
* https://github.com/kyma-project/test-infra/pull/4908

**Related issue(s)**
* https://github.com/kyma-project/kyma/issues/13305

### CODE REVIEW

File: tests/fast-integration/telemetry-test/mockserver-resources.yaml [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        app: mockserver
+      name: mockserver
+    spec:
+      containers:
+      - env:
+        - name: LOG_LEVEL
+          value: INFO
+        - name: SERVER_PORT
+          value: "1080"
+        image: mockserver/mockserver:mockserver-5.11.2
```
Comment (skhalash) [inline tests/fast-integration/telemetry-test/mockserver-resources.yaml:72]: Please, use the migrated image from our registry

Comment (lindnerby) [reply tests/fast-integration/telemetry-test/mockserver-resources.yaml]: Of course. Can you check if locator is fine now?

**Review by skhalash — APPROVED** [review]

**Pull request merged**